examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py

```diff
@@ -196,20 +209,34 @@ def main(core_context):
     optimizer = optim.Adadelta(model.parameters(), lr=args.lr)
 
     scheduler = StepLR(optimizer, step_size=1, gamma=args.gamma)
-    for epoch in range(1, args.epochs + 1):
 
-        steps_completed = epoch * len(train_loader)
-
-        train(args, model, device, train_loader, optimizer, epoch, core_context)
-        test(args, model, device, test_loader, epoch, core_context, steps_completed=steps_completed)
+    # NEW: Resume training from epochs_completed. This is useful in the case of pausing and resuming an experiment.
+    for epoch_idx in range(epochs_completed, args.epochs + 1):
```

Comment:
you have an off-by-one here due to the lingering `args.epochs + 1`, even though you are starting at 0 now